tests/assert_pass/assignments_1.yul.smt2

```diff
@@ -19,28 +19,28 @@
 (declare-fun _keccak256_32_2066_0 ((_ BitVec 256)) (_ BitVec 256))
 (declare-fun _keccak256_64_2067_0 ((_ BitVec 256) (_ BitVec 256)) (_ BitVec 256))
 (declare-fun _keccak256_2068_0 ((Array (_ BitVec 256) (_ BitVec 8)) (_ BitVec 256) (_ BitVec 256)) (_ BitVec 256))
-(define-fun _stop_flag_2069_0 () (_ BitVec 256) #x0000000000000000000000000000000000000000000000000000000000000000)
-(define-fun _revert_flag_2070_0 () (_ BitVec 256) #x0000000000000000000000000000000000000000000000000000000000000000)
+(define-fun _stop_flag_2069_0 () (_ BitVec 256) (_ bv0 256))
```

Comment:
`bv0` seems undefined?

Reply:
This is the weird SMT syntax to define bit vectors literals using decimal notation.

Reply:
lol